systemd/escape.go

```diff
@@ -0,0 +1,66 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
we can live with it here for a bit until the archive catches up, but long run it'll be less work to reuse :-)